checador/app/views/companies/_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= form_with(model: company) do |form| %>
+  <% if company.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(company.errors.count, "error") %> prohibited this company from being saved:</h2>
+
+      <ul>
+        <% company.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="field">
+    <%= form.label :name %>
+    <%= form.text_field :name %>
+  </div>
+
+  <div class="field">
+    <%= form.label :addres %>
```

Comment:
And also here...

Reply:
Thank you, Juan Carlos!
We're going to change it!

Reply:
ok

Reply:
Ready Thank you very much we are attentive to any changes